Gemfile.jruby-3.1.lock.release

```diff
@@ -805,11 +805,11 @@ GEM
       json
       simplecov
     simplecov_json_formatter (0.1.4)
-    sinatra (4.1.1)
+    sinatra (4.2.1)
       logger (>= 1.6.0)
       mustermann (~> 3.0)
       rack (>= 3.0.0, < 4)
-      rack-protection (= 4.1.1)
+      rack-protection (= 4.2.1)
```

Comment:
It looks like `rack-protection` also aligns with sinatra version. I have compared with main Gemlock.file that other sub-dependencies like base64, logger, rack boundaries do not change (make sure we don't get big changes besides our intention).